src/components/style/popover.css

```diff
@@ -0,0 +1,2 @@
+@import "./tippy/popover-default-theme.css";
+@import "./tippy/popover-unthemed-theme.css";
```

Comment:
Did exactly what we discussed: created themes, and then component css imports the theme(s) it needs.